embassy-stm32f4/src/can.rs

```diff
@@ -12,6 +12,7 @@ use embassy::util::InterruptFuture;
 use nb;
 use nb::block;
 
+#[cfg(not(any(feature = "stm32f401", feature = "stm32f410", feature = "stm32f411",)))]
```

Comment:
I think it's simpler to cfg-out the `mod can` in lib.rs

Reply:
Yeah, I even did that with QEI, heh